csrc/logical_domain_map.cpp

```diff
@@ -115,24 +115,6 @@ std::pair<std::unordered_set<IterDomain*>, bool> getNonMappingDomainInfo(
       non_mapping_ids.insert(iaop->getIndexingIDOfValue());
       has_consumer_id = true;
     }
-  } else if (auto* top = dynamic_cast<TopKOp*>(consumer_tv->definition());
```

Comment:
Now that the extent change is represented as a resize, this special-case handling is no longer necessary